src/cargaDados/CargaDados.java

```diff
@@ -0,0 +1,29 @@
+package cargaDados;
+
+import dominio.TipoUsuario;
+import repository.ExameRepository;
+import repository.UsuarioRepository;
+
+public class CargaDados{
+    private UsuarioRepository usuarioRepository;
+    private ExameRepository exameRepository;

```

Comment:
Precisamos de uma forma de inicializar essas classes. Um construtor seria uma boa.

CargaDados(UsuarioRepository usuarioRepository, ExameRepository exameRepository) {
 this.usuarioRepository = usuarioRepository;
this.exameRepository = exameRepository;
}

Reply:
Adicionei o construtor como foi solicitado.